main/handlers/docs.go

```
@@ -115,167 +93,105 @@ func HandleLoadDocsMarkdown(logger *log.Logger, pgxConn *pgxpool.Pool, supabaseU
 
-	chunksData := []types.ChunkData{}
-	for _, chunk := range chunks {
-		chunksData = append(chunksData, types.ChunkData{
-			Text:       chunk.Text,
-			SourceURL:  chunk.Metadata.SourceURL,
-			ChunkPath:  chunk.Metadata.ChunkPath,
-			ChunkIndex: chunk.Metadata.Index,
-		})
+		// Write the pages to the response
+		w.Header().Set("Content-Type", "application/json")
+		helpers.Encode(w, r, http.StatusOK, pages)
```

golangci-lint (1.64.8)

Error return value of `helpers.Encode` is not checked

(errcheck)